src/pages/editor/RrsAndFileImportSettings.tsx

```diff
@@ -7,36 +7,35 @@ import {
   FormControlLabel,
   FormLabel,
   InputAdornment,
+  Radio,
+  RadioGroup,
   TextField,
   Typography,
 } from '@mui/material';
 import { useMutation } from '@tanstack/react-query';
 import { Field, FieldProps, Form, Formik, FormikProps } from 'formik';
-import { useTranslation } from 'react-i18next';
+import { Trans, useTranslation } from 'react-i18next';
 import { useDispatch, useSelector } from 'react-redux';
 import { updateCustomerInstitution } from '../../api/customerInstitutionsApi';
-import { StyledRightAlignedWrapper } from '../../components/styled/Wrappers';
 import { setCustomer } from '../../redux/customerReducer';
 import { setNotification } from '../../redux/notificationSlice';
 import { RootState } from '../../redux/store';
 import { CustomerInstitution, CustomerRrsType } from '../../types/customerInstitution.types';
 import { dataTestId } from '../../utils/dataTestIds';
 
-export const RightsRetentionStrategySettings = () => {
+export const RrsAndFileImportSettings = () => {
   const dispatch = useDispatch();
   const { t } = useTranslation();
   const customer = useSelector((store: RootState) => store.customer);
 
-  const updateRightsRetentionStrategy = useMutation({
+  const updatePublishStrategy = useMutation({
```

Comment:
Denne oppdaterer `customer`, og publiseringsstrategi er vel det eneste på denne siden den faktisk ikke endrer, da det skjer i annen hook i `PublishStrategySettings.tsx`
```suggestion
  const updateCustomer = useMutation({
```